### PR TITLE
Fixed deprecation warnings in materialized-views project by updating project structure.

### PR DESCRIPTION
**Pull request by Luttik**
Currently running DBT, while having this project installed as a dependency, will yield the warning:

```
13:33:39  Running with dbt=1.0.3
13:33:39  [WARNING]: Deprecated functionality
The `source-paths` config has been renamed to `model-paths`. Please update your
`dbt_project.yml` configuration to reflect this change.
13:33:39  [WARNING]: Deprecated functionality
The `data-paths` config has been renamed to `seed-paths`. Please update your
`dbt_project.yml` configuration to reflect this change.
```

This pull request makes the suggested changes and is therefore in-line with modern DBT. And in doing so will resolve the source of these warnings.

I have tested this PR on my local repositories.

### CODE REVIEW

**Comment by Luttik** [comment]
@jtcohen6 Since you're usually quick and helpful when I tag you I'll do so again. This was again fairly low-hanging fruit.

File: materialized-views/dbt_project.yml [hunk 1]
```diff
@@ -3,10 +3,10 @@ version: '0.2.0'
 config-version: 2
 require-dbt-version: ">=0.18.0"
```
Comment (jtcohen6) [inline materialized-views/dbt_project.yml]: Given that the new configs are only supported in v1.0+, could you bump this as well?
```
require-dbt-version: ">=1.0.0"
```

**Review by jtcohen6 — CHANGES_REQUESTED** [review]
Thanks @Luttik! One change requested

File: materialized-views/dbt_project.yml (hunk 1 above)
Comment (Luttik) [reply materialized-views/dbt_project.yml]: Ah that is a very relevant change. I've implemented this.

**Comment by Luttik** [comment]
@jtcohen6 Could you check it out again? I've implemented your requested changes a while ago.

**Review by jtcohen6 — APPROVED** [review]
@Luttik Sorry for the delay here! LGTM

**Pull request merged**